portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -78,9 +94,35 @@ public void entityMarshall(Entity entity) {
    *
    * @param entity the entity to unmarshall data from.
    */
-  public void entityUnMarshall(Entity entity) {
+  public void entityUnmarshall(Entity entity) {
```

Comment:
No reason to keep this function anymore. Just inline its contents into the private constructor.

Reply:
Will change in this PR.